docker-compose-with-temporal.yml

```diff
@@ -41,6 +36,7 @@ services:
     restart: on-failure
     volumes:
       - ./keys:/root/.ssh
+      - ./keys:/retool_backend/keys
       - ssh:/retool_backend/autogen_ssh_keys
       - ./retool:/usr/local/retool-git-repo
       - ${BOOTSTRAP_SOURCE:-./retool}:/usr/local/retool-repo
```

Comment:
I think we can drop all these, but can do that later

(`backend/docker_scripts/generate_key_pair.sh` is deprecated)